src/NUnitFramework/framework/Internal/Tests/ParameterizedFixtureSuite.cs

```diff
@@ -42,6 +42,16 @@ public ParameterizedFixtureSuite(ITypeInfo typeInfo) : base(typeInfo.Namespace,
             _genericFixture = typeInfo.ContainsGenericParameters;
         }
 
+        /// <summary>
+        /// Copy constructor style to create a filtered copy of the given parameterized fixture suite
```

Comment:
I want to move in the direction of using end marks (period, question mark or colon + list) for all sentences in user-facing messages.

You'd call this a 'copy-style constructor.' 'Copy constructor style' doesn't mean anything to me. Maybe:

```suggestion
        /// Creates a copy of the given suite with only the descendants that pass the specified filter.
```

(Same comment for all occurrences.)